marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -254,6 +380,17 @@ private void verifyIndexes(String dbName) {
         assertTrue(pathExpressions.containsAll(Arrays.asList("//*:instance/testPathIndexForDHFPROD4704", "/(es:envelope|envelope)/(es:instance|instance)/Customer/someOtherProperty")));
     }
 
+    private void updateIndexes() {
```

Comment:
This should have a more self-documenting name - it's not just updating the indexes, it's removing the path and element indexes. But - I think we only care about element indexes here. So this should only clear about the element indexes, and it should have a name that makes it clear that's what it does - i.e. "removeRangeElementIndexesFromFinalDatabase()".